### PR TITLE
Update missing voyages enum values in voyages timeseries the sdk

### PR DESCRIPTION
**Pull request by migueluzcategui**
#### RELATED TICKETS

- TBD

#### RELATED PRs

- NA

#### CHANGELOG

- Add missing property in documentation
- Handle when there's no breakdown property
- Removing flaky tests and flaky hardcoded assertions

#### TESTS

- describe your test strategy

#### COMMENTS

- any other business

### CODE REVIEW

File: tests/endpoints/test_onshore_inventories_search.py [hunk 1]
```diff
@@ -13,26 +13,6 @@ def test_timeseries_returns_data_frame(self):
 
         assert len(df) > 100
 
-    def test_filter_by_measurement_ids(self):
```
Comment (migueluzcategui) [inline tests/endpoints/test_onshore_inventories_search.py:16]: Removing flaky test

File: tests/endpoints/test_voyages_search_enriched.py [hunk 2]
```diff
@@ -98,10 +98,13 @@ def test_search_from_description_raw(self):
         )
 
         assert len(res) > 0
-        assert res[0].schema_version == "1.0.0"
         assert res[0].vessel is not None
         assert len(res[0].events) > 0
         assert res[0].events[0].event_group is not None
+        assert res[0].start_timestamp is not None
```
Comment (migueluzcategui) [inline tests/endpoints/test_voyages_search_enriched.py:104]: Adding assert for the properties that should never be nullable, and removing the hardcoded schema version.

File: vortexasdk/endpoints/voyages_timeseries_v2.py [hunk 3]
```diff
@@ -29,19 +31,21 @@ class VoyagesTimeseriesV2(Search):
...
+        breakdown_property: Defines the voyage property to aggregate upon. Can be one of: `'status'`, `'vessel-class-legacy'`, `'vessel-class-group'`, `'vessel-class-coarse'`, `'vessel-class-granular'`, `'origin-region'`, `'origin-shipping-region'`, `'origin-trading-region'`, `'origin-trading-sub-region'`, `'origin-trading-block'`, `'origin-country'`, `'origin-port'`, `'origin-terminal'`, `'origin_port_or_sts_zone'`, `'destination-region'`, `'destination-shipping-region'`, `'destination-trading-region'`, `'destination-trading-sub-region'`, `'destination-trading-block'`, `'destination-country'`, `'destination-port'`, `'destination-terminal'`, `'destination_port_or_sts_zone'`, `'location-waypoint'`, `'product-group'`, `'product-group-product'`, `'product-category'`, `'product-grade'`, `'corporate-entity-cargo-trader'`, `'corporate-entity-charterer'`, `'corporate-entity-effective-controller'`, `'corporate-entity-vessel-owner'`, `'corporate-entity-time-charterer'`.
```
Comment (samhuk) [inline vortexasdk/endpoints/voyages_timeseries_v2.py:36]: Are some missing? https://vortexa.slack.com/archives/CD5MA408P/p1726580430912939?thread_ts=1726580034.870479&cid=CD5MA408P

**Review by samhuk — CHANGES_REQUESTED** [review]
There's more splits in the UI (38) than in the python code (33). EVerything else LGTM :)

File: vortexasdk/endpoints/voyages_timeseries_v2.py (hunk 3 above)
Comment (migueluzcategui) [reply vortexasdk/endpoints/voyages_timeseries_v2.py:36]: I added all the properties supported in the API, including the ones disabled from the UI. I also removed some the legacy vessel class.

**Review by samhuk — APPROVED** [review]

**Pull request merged**